discovery-provider/src/api/v1/reactions.py

```diff
@@ -0,0 +1,50 @@
+from flask_restx import Namespace, Resource, fields, marshal_with, reqparse
+from src.api.v1.helpers import (
+    DescriptiveArgument,
+    extend_reaction,
+    make_response,
+    success_response,
+)
+from src.api.v1.models.reactions import reaction
+from src.queries.reactions import get_reactions
+from src.utils.db_session import get_db_read_replica
+from src.utils.redis_cache import cache
+from src.utils.redis_metrics import record_metrics
+
+ns = Namespace("reactions", description="Reaction related operations")
+
+get_reactions_parser = reqparse.RequestParser(argument_class=DescriptiveArgument)
+get_reactions_parser.add_argument(
+    "type", required=True, description="The type of reactions for which to query."
+)
+get_reactions_parser.add_argument(
+    "tx_id",
+    required=True,
+    action="append",
+    description="The `reacted_to` transaction id(s) of the reactions in question.",
+)
+
+get_reactions_response = make_response(
+    "reactions", ns, fields.List(fields.Nested(reaction))
+)
+
+
+@ns.route("", doc=False)
```

Comment:
Note that `doc=False` will mean that libs/sdk won't get autogenerated client for this

Reply:
v1/full it goes